### PR TITLE
QA-451: testing: implement san logfile reading in testing.js

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

Manage server *san logfiles through testing.js
- read the *SAN configurations from the environment on start
- When launching the servers, prepare an individual SAN-environment that puts the logfiles in the servers respective directory.
- on shutdown, check for these files, if they are there utilize the CRASH-mechanismus to represent their result to the testresults.

San reports written by the `testing.js`-`arangosh` itself (and thus executed test-files) will not be recognized by this PR.

- [x] :pizza: New feature
- [ ] Backports
  - [ ] Backport for 3.11: *(Please link PR)*
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/QA-451

### CODE REVIEW

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/job/arangodb-ANY-linux-san.x86-64/1949/ *san run

**Review by markuspf — APPROVED** [review]
I think this will make it slightly easier to look at failed *SAN tests.

I can't really judge though, because the test run is gone already. Once we have a failing SAN run again that I look at, I will try to give constructive ways of improving the output further.

File: tests/test-definitions.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 
 # Single Server only Tests
-BackupAuthNoSysTests,BackupAuthSysTests,BackupNoAuthSysTests,arangosh,arangobench,permissions,paths_server,version,active_failover single name=single_server_only
+BackupAuthNoSysTests,BackupAuthSysTests,BackupNoAuthSysTests,arangosh,arangobench,permissions,paths_server,version,active_failover single name=single_server_only -- --extremeVerbosity true
```
Comment (mpoeter) [inline tests/test-definitions.txt:7]: Is this intentional or just some leftover?

**Review by mpoeter — APPROVED** [review]

File: tests/test-definitions.txt (hunk 1 above)
Comment (dothebart) [reply tests/test-definitions.txt]: ah, debug. reverted.

**Pull request merged**